srcpkgs/slirp4netns/template

```diff
@@ -10,8 +10,12 @@ maintainer="Cameron Nemo <cnemo@tutanota.com>"
 license="GPL-2.0-only"
 homepage="https://github.com/rootless-containers/slirp4netns"
 distfiles="https://github.com/rootless-containers/slirp4netns/archive/v${version}.tar.gz"
-checksum=5c9e4dd7d99683a0c9ba8509aec865245604db83359ff1b7c99f9f3ae1569a28
+checksum=5ff0d3e4bf6b11c8a4fcf5bd3219b8d52096e3b8cc73ca760aa554bb1eb08768
 
 pre_configure() {
 	./autogen.sh
 }
+
+do_check() {
+	: # tests fail due to use of unshare (unavailable with chroot util-linux)
+}
```

Comment:
This should be using `make_check=no` or maybe `make_check=extended` if it works when ran locally.

Reply:
done